lib/ProductOpener/Ingredients.pm

```diff
@@ -1625,11 +1627,20 @@ sub parse_ingredients_text ($product_ref) {
 					}
 
 					# sel marin (France, Italie)
-					# -> if we have origins, put "origins:" before
-					if (    ($between =~ $separators)
+					# -> if we have origins, put "origins:" before or "製造" at the end for Japanese
```

Comment:
That should not be needed, "origins: [some origin in Japanese]" should be recognized later I think.